shared/src/persistence/elasticsearch/fetchPendingItems.js

```diff
@@ -73,6 +73,18 @@ exports.fetchPendingItems = async ({
         },
       },
       size,
+      sort: [
```

Comment:
Nit: does this file need some unit test coverage?

Reply:
Oooo, good find.  Yes.  Yes it should.

Reply:
Added a unit test to cover the sort lines!